src/config/SerializationConfig.ts

```diff
@@ -26,6 +26,18 @@ export interface SerializationConfig {
 
     /**
      * Defines how the `number` type is represented on the cluster side. By default, it is serialized as `Double`.
+     * This option can be one of the following strings. The casing of the string does not matter, it's converted to lower
+     * case internally.
+     * * `byte`(8-bit signed integer)
```

Comment:
How is this rendered? (the `*`). IIRC, we should be using `<ul><li>elem1</li><li>elem2</li></ul>` like syntax for lists

Reply:
try and see :) markdown, for typedoc